frontend/src/app.css

```diff
@@ -47,38 +63,26 @@
   #year-list .ais-RefinementList-list {
     columns: 5rem auto;
   }
-  .seek-track-slider .track-slider::-webkit-slider-thumb {
-    @apply -mt-1 w-3.5 h-3.5 bg-green-800 rounded-full appearance-none cursor-pointer;
+  .seek-slider input[type='range'] {
+    &::-webkit-slider-thumb {
+      @apply -mt-1 w-3.5 h-3.5;
+    }
   }
-  .seek-track-slider .track-slider::-webkit-slider-runnable-track {
-    @apply h-1.5;
-  }
-  .seek-volume-slider .volume-slider::-webkit-slider-thumb {
-    @apply -mt-0.5 w-2.5 h-2.5 bg-green-800 rounded-full appearance-none cursor-pointer;
-  }
-  .seek-volume-slider .volume-slider::-webkit-slider-runnable-track {
-    @apply h-1.5;
+  .volume-slider input[type='range'] {
+    &::-webkit-slider-thumb {
+      @apply -mt-0.5 w-2.5 h-2.5;
+    }
   }
 
   /* styles for Firefox */
-  .seek-track-slider .track-slider::-moz-range-thumb {
-    @apply -mt-1 w-3.5 h-3.5 bg-green-800 rounded-full appearance-none cursor-pointer;
-  }
-  .seek-track-slider .track-slider::-moz-range-track {
-    @apply h-1.5;
-  }
-  .seek-volume-slider .volume-slider::-moz-range-thumb {
-    @apply -mt-0.5 w-2.5 h-2.5 bg-green-800 rounded-full appearance-none cursor-pointer;
-  }
-  .seek-volume-slider .volume-slider::-moz-range-track {
-    @apply h-1.5;
-  }
-}
-body {
-  height: calc(100vh - 108px) !important;
-}
-@media (max-width: 640px) {
-  body {
-    height: calc(100vh - 200px) !important;
+  .seek-track-slider input[type='range'] {
+    &::-moz-range-thumb {
+      @apply -mt-1 w-3.5 h-3.5;
+    }
+  }
+  .seek-volume-slider input[type='range'] {
+    &::-moz-range-thumb {
+      @apply -mt-0.5 w-2.5 h-2.5;
+    }
```

Comment:
Don't make the thumb of different size. Let them be of the same size, and add sizing to the base style above.

Reply:
I will do the same but it will look ugly because the height and width of the sleek and volume sliders are different, otherwise it's a bid deal to make same sizes

Reply:
Height can be the same, that's what I'm speaking about.